src/SFA.DAS.Roatp.CourseManagement.Api.UnitTests/Controllers/ProviderCourseEditControllerTests/UpdateProviderCourseContactDetailsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Roatp.CourseManagement.Api.Controllers;
+using SFA.DAS.Roatp.CourseManagement.Application.Standards.Commands.UpdateContactDetails;
+using SFA.DAS.Testing.AutoFixture;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Roatp.CourseManagement.Api.UnitTests.Controllers.ProviderCourseEditControllerTests
+{
+    [TestFixture]
+    public class UpdateProviderCourseContactDetailsTests
+    {
+        [Test, MoqAutoData]
+        public async Task UpdateProviderCourseContactDetails_Success_ReturnsNoContent(
+            [Frozen] Mock<IMediator> mediator,
+            [Greedy] ProviderCourseEditController sut,
+            int ukprn, int larsCode, UpdateContactDetailsCommand command)
+        {
+            mediator.Setup(m => m.Send(It.Is<UpdateContactDetailsCommand>(c => c.Ukprn == ukprn && c.LarsCode == larsCode), It.IsAny<CancellationToken>())).ReturnsAsync(HttpStatusCode.NoContent);
+
+            var result = await sut.UpdateProviderCourseContactDetails(ukprn, larsCode, command);
+
+            (result as NoContentResult).Should().NotBeNull();
+        }
+
+        [Test, MoqAutoData]
```

Comment:
Can you add the same test with a 'InternalServerError code too

Id suggest using TestCase() but you can also do it as a separate test.  Ping me if you want to discuss